plugins/jobs/web_client/JobStatus.js

```diff
@@ -41,42 +45,55 @@ var JobStatus = {
             this[name] = info.value;
             this._map[info.value] = {
                 text: info.text,
-                icon: info.icon
+                icon: info.icon,
+                color: info.color
             };
         }, this);
+    },
+
+    getAllStatus: function () {
```

Comment:
This can be called `getAllStatuses` or just `getAll`. Also, I think this can just be `return _.values(this._map)`.